backend/vapid.js

```diff
@@ -0,0 +1,91 @@
+import sbp from '@sbp/sbp'
+
+let vapidPublicKey: string
+let vapidPrivateKey: Object
+
+// TODO: Load from configuration
+const vapid = { VAPID_EMAIL: 'mailto:test@example.com' }
```

Comment:
As the comment says, the value should be configurable. This technically can also be an HTTPS URL, but should be a way to get in contact. It is part of VAPID, which is used to restrict who can send a notification. See, e.g., <https://blog.mozilla.org/services/2016/04/04/using-vapid-with-webpush/>.

Reply:
Forgot to add, the email is supposed to be there so that, e.g., Apple / Google / Microsoft / Mozilla can get in touch with us about issues. In practice, I doubt it's ever used, apart from maybe automated notifications.

Do note that with the previous work on push notifications, we also had this, with this exact name, but it was `import`ed from a JSON file.